api/v1alpha1/openapi.yaml

```diff
@@ -24,11 +24,26 @@ tags:
     description: Operations on EnrollmentRequest resources.
   - name: fleet
     description: Operations on Fleet resources.
+  - name: information
+    description: Operations for receiving service information.
   - name: repository
     description: Operations on Repository resources.
   - name: resourcesync
     description: Operations on ResourceSync resources.
 paths:
+  /api/v1/info/version:
+    get:
+      tags:
+        - information
+      description: Get service version.
+      operationId: version
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Version'
```

Comment:
Add all of the 4XX errors that other API definitions have

Reply:
there are no no-200 results, there're no point of failure, we just propagate results